.github/workflows/ci-aqua-security-trivy-tests.yml

```diff
@@ -0,0 +1,30 @@
+name: ci-aqua-security-trivy-tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: "*/30 * * * *"
+jobs:
+  build:
+    name: trivy-tests
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Build Docker image
+        run: |
+          make image-local
```

Comment:
@kevinmingtarja this is how we build our docker image, and this step references a dockerfile as detailed here https://github.com/dgraph-io/dgraph/blob/skrdgraph/docker_sec_checks/Makefile#L76 ... which points to https://github.com/dgraph-io/dgraph/blob/skrdgraph/docker_sec_checks/contrib/Dockerfile

the GH-action in this PR checks for security CVEs using `trivy` and dumps the output into https://github.com/dgraph-io/dgraph/security/code-scanning (where we can analyze the severity, and make fixes)

This GH Action captures the exact CVEs.